src/hb-ot-post-table.hh

```diff
@@ -77,6 +119,60 @@ struct post
     return_trace (true);
   }
 
+  inline bool get_glyph_name (hb_codepoint_t glyph,
+                              char *buffer, unsigned int buffer_length,
+                              unsigned int blob_len) const
+  {
+    if (version.to_int () == 0x00010000)
+    {
+      if (glyph >= NUM_FORMAT1_NAMES)
+        return false;
+
+      strncpy(buffer, format1_names[glyph], buffer_length);
+      buffer[buffer_length] = '\0';
```

Comment:
Ok I see you have this.  But it should be `buffer_length - 1`, no?  I prefer checking for zero here as well.  Maybe just here, not in the caller.